src/app/dashboard/accounts/page.tsx

```diff
@@ -19,9 +20,6 @@ import Onboarding from '@/components/onboarding/Onboarding';
 import { useAccounts } from '@/hooks/api';
 import mapAccounts from '@/helpers/mapAccounts';
 
-const NetWorthHistogram = dynamic(() => import('@/components/pages/accounts/NetWorthHistogram'), { ssr: false });
-const IncomeExpenseHistogram = dynamic(() => import('@/components/pages/accounts/IncomeExpenseHistogram'), { ssr: false });
```

Comment:
unrelated but since we don't use zoom/pan in these components anymore, this is not needed.